src/fssrgen/FsSrGen.fsproj

```diff
@@ -1,60 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why the delete?  Don't we still have to build these for .NET Framework?  Or is the tools fully .NET-Standard compliant and usable on either?